### DISCUSSION TITLE
Nuclei stats not working

### DISCUSSION THREAD

**Discussion by dan-rd** [projectdiscovery/nuclei#3371] [Q&A] [answered]
I am trying to get the stats output in a json file to monitor the nuclei scan and keep a record of how long it took to complete the scan. But I am not able to make it work.

The command I used is: 
`nuclei -u <domain> -o nuclei.json -json -stats -sj`

Am I missing something? Thanks for your help :)

![image](https://user-images.githubusercontent.com/91903112/221602646-6fd10444-0688-4dee-a6d6-133846af4250.png)

**Comment by olearycrew** [comment, marked as answer]
It looks like some of your templates may have syntax errors? Other than that it appears to be working correctly...also take a look at the updated nuclei as there are now options for exporting results vs. writing JSON lines to the output
